ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1741,4 +1741,8 @@ router bgp {{ router_bgp.as }}
       recovery delay {{ session_tracker.recovery_delay }} seconds
 {%         endif %}
 {%     endfor %}
+{%     if router_bgp.eos_cli is arista.avd.defined %}
+   !
+   {{ router_bgp.eos_cli | indent(6, false) }}
+{%     endif %}
```

Comment:
you can add a molecule test for this feature.

Reply:
Added the test.